lib/Models/Terria.ts

```diff
@@ -561,6 +574,7 @@ export default class Terria {
         !terriaError.raisedToUser)
     ) {
       terriaError.raisedToUser = true;
+      this.errorService.error(terriaError);
       this.error.raiseEvent(terriaError);
     } else {
       console.log(terriaError);
```

Comment:
I think we should call `this.errorService.error(terriaError)` regardless of whether the error is shown to the user

Reply:
Have changed this to log to service for every call.